### ISSUE TITLE
Support RedisCache

### ISSUE THREAD

**Issue by prajon84** [thomasgalliker/HttpClient.Caching#12]
Any directions / effort for support of redis cache?

**Comment by thomasgalliker** [comment]
Not really. Do you have sample code of such an integration? Unfortunately, I dont use Redis in any of my projects.

**Comment by thomasgalliker** [comment]
I might have a look at it once in the future. Thanks for the link about Distributed Caching.
